usaon_vta_survey/models/tables.py

```diff
@@ -213,14 +226,6 @@ class ResponseObservingSystem(BaseModel, IORelationshipMixin):
         'polymorphic_on': type,
     }
```

Comment:
`id` and `response_id` are also common fields. Can those go in the mixin as well?

Reply:
Sqlalchemy did not like when i tried to do this.